<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentasWindow</class>
 <widget class="QWidget" name="VentasWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblCliente">
       <property name="text">
        <string>Cliente:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboCliente"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablaProductos">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Código</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Precio</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAgregarProducto">
     <property name="text">
      <string>Agregar producto</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tablaVenta">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>Código</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Precio</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ventaButtonsLayout">
     <item>
      <widget class="QPushButton" name="btnEliminarProducto">
       <property name="text">
        <string>Eliminar producto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTerminarVenta">
       <property name="text">
        <string>Terminar venta</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblTotalVenta">
     <property name="text">
      <string>Total: $0.00</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
